default/stringtables/en.txt

```diff
@@ -1187,6 +1187,12 @@ Sets how fast to render dots for fleet supply lines.
 OPTIONS_DB_FLEET_EXPLORE_IGNORE_HOSTILE
 Ignore hostile ships when a fleet is set to explore
 
+OPTIONS_DB_FLEET_EXPLORE_SYSTEM_ROUTE_LIMIT
+Max number of fleet routes used to determine favored route to an unexplored system. A value of 0 or less indicates no limit.
```

Comment:
This explanation lacks context. If it's supposed to ever be read by a player, it needs some elaboration...

Reply:
please rewrite this text in a later commit...